src/lib/frontend/frontend.ml

```diff
@@ -276,8 +276,8 @@ module Make(SAT : Sat_solver_sig.S) : S with type sat_env = SAT.t = struct
       Printer.print_status_unsat ~validity_mode
         (Some loc) (Some time) (Some steps) (get_goal_name d);
       if get_unsat_core() &&
-         not (get_debug_unsat_core()) &&
-         not (get_save_used_context())
+         not (get_debug_unsat_core())
+         (* && not (get_save_used_context()) *)
```

Comment:
Why removing this test ? it's here to not print the UNSAT core if the user asked to save the used context (name of axioms) in a `.used` file.

Reply:
This has been revert since we do not use save_used_context anymore